### DISCUSSION TITLE
❓ Questions / Help / Support

### DISCUSSION THREAD

**Discussion by retyer** [snakers4/silero-models#150] [Q&A] [answered]
## ❓ Questions and Help

Hi,
Is it possible to provide examples of how to containerize your models as a service? Is it possible to containerize your models using torchserve?

A detailed answer would be highly appreciated, if possible.

**Comment by snakers4** [comment, marked as answer]
Hi,

Containerization is definitely possible, but we offer this only commercially.

As for `torchserve` - we do not use it,  most likely for real production usage it will be heavily limited architecture-wise.

**Reply by retyer** [reply]
Thank you for the answer!
